content/ui-patterns/forms.mdx

```diff
@@ -0,0 +1,382 @@
+---
+title: Forms
+---
+
+import {Box, Heading, Link, Text} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Forms are used to complete tasks that require data input from the user. For example: creating a new repo, configuring settings, and logging in.
+</Box>
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Primer's form design guidelines aim to minimize the effort and cognitive load required to complete a task that involves a form.
+</Box>
+
+## Form control anatomy
+
+![diagrams labeling the anatomy of a text field and a checkbox field](https://user-images.githubusercontent.com/2313998/164065184-e3c615a3-fb68-4878-b7f8-9a57efb748cb.png )
```

Comment:
I was a bit confused at first looking at this example, because my immediate reaction wasn't thinking that this was a vertical vs. horizontal layout, but rather two different form types (text input vs. radio check). Do we need to classify this as horizontal vs. vertical, or is it enough to describe the different parts of the forms (label, input, ...)? 🤔

Reply:
Thanks for calling that out. I don't think we need to classify horizontal vs vertical. I see how it's confusing.